packages/env/lib/config.js

```diff
@@ -257,6 +257,21 @@ function parseSourceString( sourceString, { workDirectoryPath } ) {
 		};
 	}
 
+	const zipFields = sourceString.match(
+		/^https?:\/\/([^\s$.?#].[^\s]*)\.zip$/
+	);
+	if ( zipFields ) {
+		return {
+			type: 'zip',
+			url: sourceString,
+			path: path.resolve(
+				workDirectoryPath,
+				encodeURIComponent( zipFields[ 1 ] )
+			),
+			basename: encodeURIComponent( zipFields[ 1 ] ),
```

Comment:
Does having `%` in the filenames blow up on Windows?

Reply:
https://stackoverflow.com/a/31976060

It should be fine.